tests/unit/test.errors.js

```diff
@@ -5,7 +5,7 @@ import {assert} from 'chai';
 import {
   onlyErrorsWithCode, isErrorWithCode, onlyInstancesOf,
 } from '../../src/errors';
-import {makeSureItFails} from './helpers';
+import {makeSureItFails, ErrorWithCode} from './helpers';
```

Comment:
If we do not use sinon for the new tests, these changes related to `ErrorWithCode` are not needed anymore, but I'm not strongly against them.